docs/diagrams/ListEntryActivityDiagram.puml

```diff
@@ -0,0 +1,16 @@
+@startuml
```

Comment:
i do not see your activity diagram in developer guide at all. did you forget to include it?

Reply:
The activity diagram is wrong, so I did not include it in the DG.